src/CozyUtils.js

```diff
@@ -129,6 +130,20 @@ class CozyUtils {
   save(params) {
     return this.client.save(params)
   }
+
+  async deleteAccount(accountId) {
+    const accountsCollection = this.client.collection(DOCTYPE_ACCOUNT)
+    const response = await accountsCollection.get(accountId)
+    const doc = get(response, 'data')
+    if (!doc && process.env.NODE_ENV === 'development') {
+      log(
+        'info',
+        "Coudln't find io.cozy.accounts in development mode, skipping deleting the account."
```

Comment:
I think we should put as much as info we can in the log. "Coudln't find io.cozy.account for accountId : ${accountId}"

Reply:
typo : Couldn't